src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py

```diff
@@ -50,7 +50,11 @@ def flexible_server_create(cmd, client,
     # Generate missing parameters
     location, resource_group_name, server_name = generate_missing_parameters(cmd, location, resource_group_name,
                                                                              server_name, 'postgres')
+
     server_name = server_name.lower()
+    if high_availability and high_availability.lower() == 'enabled':
+        logger.warning('\'Enabled\' value for high availability parameter will be deprecated. Please use \'ZoneRedundant\' or \'SameZone\' instead.')
```

Comment:
I can't find a way to deprecate only a single possible value for the argument. The closest I could do was this, but the customer will be confused because the whole command `--high-availability` appears as deprecated when it's not:
![image](https://user-images.githubusercontent.com/108426424/185858342-ca2ae0e4-5aaf-4cb7-a789-741dc015fbf4.png)

Reply:
I think warning log is fine since it's not deprecation for the whole parameter but for some enum value of this parameter